src/test/java/io/appium/java_client/ios/IOSAlertTest.java

```diff
@@ -16,25 +16,36 @@
 
 package io.appium.java_client.ios;
 
-import static org.openqa.selenium.support.ui.ExpectedConditions.alertIsPresent;
-
 import io.appium.java_client.MobileBy;
 import org.junit.Test;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
+import static org.junit.Assert.assertNotNull;
+import static org.openqa.selenium.support.ui.ExpectedConditions.alertIsPresent;
+
 public class IOSAlertTest extends BaseIOSTest {
 
     @Test public void acceptAlertTest() {
         driver.findElement(MobileBy
             .IosUIAutomation(".elements().withName(\"show alert\")")).click();
         WebDriverWait wating = new WebDriverWait(driver, 10000);
-        wating.until(alertIsPresent()).accept();
+        wating.until(alertIsPresent());
+        driver.switchTo().alert().accept();
     }
 
     @Test public void dismissAlertTest() {
         driver.findElement(MobileBy
             .IosUIAutomation(".elements().withName(\"show alert\")")).click();
         WebDriverWait wating = new WebDriverWait(driver, 10000);
-        wating.until(alertIsPresent()).dismiss();
+        wating.until(alertIsPresent());
+        driver.switchTo().alert().dismiss();
+    }
+
+    @Test public void getAlertTextTest() {
+        driver.findElement(MobileBy
+            .IosUIAutomation(".elements().withName(\"show alert\")")).click();
+        WebDriverWait wating = new WebDriverWait(driver, 10000);
+        wating.until(alertIsPresent());
+        assertNotNull(driver.switchTo().alert().getText());
```

Comment:
@SrinivasanTarget 
I would like to advive you to
```java
assertTrue(!StringUtils.isBlank(driver.switchTo().alert().getText()));
```
What if the empty string would be returned? :)